src/Directory.Build.targets

```diff
@@ -54,4 +54,15 @@
   <Target Name="DumpTargets" BeforeTargets="ResolveProjectReferences">
     <Message Text="DumpTargets> $(OutputPath), C=[$(Configuration)], CG=[$(ConfigurationGroup)], OG=[$(OSGroup)], TG=[$(TargetGroup)]" Importance="Low" />
   </Target>
-</Project> 
+
+  <!--
+    Harvesting supported versions can sometimes lead to unknown versions when harvesting a package
+    which has placeholders for older frameworks like net45. We should just exclude those from validation
+    since we cannot easily determine what they are without looking at each specific platform itself.
+  -->
+  <Target Name="ExcludeSupportedFrameworksWithUnknownVersion" BeforeTargets="ValidateLibraryPackage">
+    <ItemGroup>
+      <SupportedFramework Remove="@(SupportedFramework)" Condition="'%(SupportedFramework.Version)' == 'unknown'" />
```

Comment:
Won't we lose coverage in those frameworks then? Like the net45 example that you mention, we could drop support for those in case we used to have a placeholder and we won't catch that any longer, right?

Reply:
There is definitely potential but I don't know of a great way to gather all the necessary versions to keep the validation.